test/verify/check-networking-firewall

```diff
@@ -138,15 +152,22 @@ class TestFirewall(MachineCase):
         m = self.machine
 
         self.login_and_go("/network/firewall")
+        b.wait_in_text("#zones-listing", "Public")
+        b.wait_in_text("#zones-listing", "default")
 
         def removeService(service):
             b.click("tr[data-row-id='{}'] .btn.pficon-delete".format(service))
+            if b.is_present("#remove-services-dialog-body #zone-input input:not(:checked)"):
+                b.click("#remove-services-dialog-body #zone-input input:not(:checked)")
+            b.click("#remove-services-dialog button.btn-primary")
             b.wait_not_present("tr[data-row-id='{}']".format(service))
 
         # click on the "Add Services" button
         b.wait_present("caption button.btn-primary:enabled")
         b.click("caption button.btn-primary")
         b.wait_present("#cockpit_modal_dialog .list-view-pf input[data-id='pop3']")
+        if m.image not in ["rhel-8-0", "rhel-8-1"]:
```

Comment:
This should come with a comment that explains what's special here. E. g. a bugzilla. Same for the other special cases below.
This also needs to be updated for rhel-8-0-distropkg.